zulip_bots/zulip_bots/test_lib.py

```diff
@@ -35,7 +35,7 @@ def get_bot_message_handler(self):
         # handler class. Eventually, we want bot's handler classes to
         # inherit from a common prototype specifying the handle_message
         # function.
-        lib_module = import_module('zulip_bots.bots.{bot}.{bot}'.format(bot=self.bot_name))
+        lib_module = import_module('zulip_bots.bots.{bot}.{bot}'.format(bot=self.bot_name))  # type: Any
```

Comment:
`types.ModuleType` again here?

Reply:
This one gives me the `Module has no attribute "handler_class"` error.